Update SSO docs (#17934)
PrefectHQ/prefect: docs/v3/manage/cloud/manage-users/configure-sso.mdx

When using SSO, Prefect Cloud won't store passwords for any accounts managed by your identity provider.
Members of your Prefect Cloud account will log in and authenticate using your identity provider.

Once your SSO integration is set up, non-admins are required to authenticate through the SSO provider when
accessing account resources.

## Configure SSO

<Note>Available for Pro and Enterprise plans.</Note>

To begin configuring SSO, open the dropdown from the top left and choose **Account settings** from the menu. Select **SSO** from the Settings menu.

Enter the domains that require users to authenticate with SSO. This is typically the domain of your organization (e.g., `prefect.io`).

> To complete the SSO configuration, click the blue **Configure** button. Copy the generated link and navigate to the Identity Provider Configuration dashboard. Select your identity provider from the list.

Reviewer: ```suggestion
To complete the SSO configuration, click **Configure**. Copy the generated link and navigate to the Identity Provider Configuration dashboard. Select your identity provider from the list.
```

![Opening the Identity Provider Configuration dashboard.](/v3/img/ui/cloud-sso-dashboard.png)

<Tip>If your SSO provider isn't listed, use the generic `SAML` or `Open ID Connect` integration instead.</Tip>

Once you complete the SSO configuration, your users will authenticate through your identity provider.

## Directory sync

<Note>Available for Enterprise plans only.</Note>

> **Directory sync** allows you to manage user access to Prefect Cloud resources through your identity provider without the need to manually provision through the Prefect Cloud UI. Directory sync uses the SCIM 2.0 API to interface with your identity provider.

Reviewer: ```suggestion
**Directory sync** allows you to manage user access to Prefect Cloud resources through your identity provider without needing to manually configure access in the Prefect Cloud UI. Directory sync uses the [SCIM 2.0 API](https://simplecloud.info/#Specification) to interface with your identity provider.
```

When a user is assigned to the Prefect Cloud application in your identity provider, their user account is automatically provisioned in Prefect Cloud.

Provisioned users are assigned basic “Member” roles and have access to any resources that role entails.

When a user is unassigned from the Prefect Cloud application in your identity provider, their access to Prefect Cloud resources is automatically removed.

> To configure Directory Sync/SCIM, click the blue **Configure** button. Copy the generated link and navigate to the Organization dashboard.

Reviewer 2: ```suggestion
To configure Directory Sync/SCIM, click **Configure**. Copy the generated link and navigate to the Organization dashboard.
```
